pkg/client/columns.go

```diff
@@ -81,3 +82,56 @@ func (c *Client) ListColumns(ctx context.Context, parentResourceID *v2.ResourceI
 
 	return ret, nextPageToken, nil
 }
+
+// If the privilege is "grant", it grants SELECT, INSERT, UPDATE, and REFERENCES privileges.
+func (c *Client) GrantColumnPrivilege(ctx context.Context, table string, column string, user string, privilege string) error {
+	userSplit := strings.Split(user, "@")
+	if len(userSplit) != 2 {
+		return fmt.Errorf("invalid user format: %s", user)
+	}
+	userGrant := fmt.Sprintf("%s'@'%s", userSplit[0], userSplit[1])
```

Comment:
please use parameterized queries

Reply:
Ok I didn't know this. I guess maybe let's do some basic escaping then. Thank you